tests/db_level_on_delete_checks/tests.py

```diff
@@ -0,0 +1,183 @@
+from django.contrib.contenttypes.fields import GenericForeignKey
```

Comment:
Please move these tests to the `tests/invalid_models_tests/test_relative_fields.py` and reuse existing models.

Reply:
Done